dhis-2/dhis-services/dhis-service-program-rule/src/main/java/org/hisp/dhis/programrule/engine/ProgramRuleEngine.java

```diff
@@ -57,6 +59,11 @@
 
     private static final String USER = "USER";
 
+    private static final String REGEX = "d2:inOrgUnitGroup\\( *(([\\d/\\*\\+\\-%\\. ]+)|" +
+        "( *'[^']*'))*( *, *(([\\d/\\*\\+\\-%\\. ]+)|'[^']*'))* *\\)";
```

Comment:
What is this regexp doing? We have a UID-regex in ExpressionService which can be reused somehow if we are looking for UIDs here.